misk/src/main/kotlin/misk/security/authz/AccessInterceptor.kt

```diff
@@ -27,9 +28,23 @@ class AccessInterceptor private constructor(
     return chain.proceed(chain.args)
   }
 
+  /** Check whether the caller is allowed to access this endpoint */
+  private fun isAuthorized(caller: MiskCaller): Boolean {
```

Comment:
I moved this from lower in the file to make it clearer that it's not part of Factory